rs/state_layout/src/state_layout.rs

```diff
@@ -2459,11 +2454,40 @@ impl TryFrom<pb_canister_state_bits::CanisterStateBits> for CanisterStateBits {
             );
         }
 
-        let on_low_wasm_memory_hook_status: Option<
-            pb_canister_state_bits::OnLowWasmMemoryHookStatus,
-        > = value.on_low_wasm_memory_hook_status.map(|v| {
-            pb_canister_state_bits::OnLowWasmMemoryHookStatus::try_from(v).unwrap_or_default()
-        });
+        let pb_task_queue: Result<pb_canister_state_bits::TaskQueue, ProxyDecodeError> =
+            try_from_option_field(value.tasks, "CanisterStateBits::tasks");
+
+        let task_queue = match pb_task_queue {
+            Ok(tasks) => TaskQueue::try_from(tasks).unwrap(),
```

Comment:
We shouldn't unwrap here I think, can we not do something like `TaskQueue::try_from(tasks)?`